framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2903,23 +2915,29 @@ protected void checkThrownExpression(ThrowTree tree) {
                 AnnotationMirrorSet foundEffective = throwType.getEffectiveAnnotations();
                 if (!atypeFactory.getQualifierHierarchy().isSubtype(foundEffective, required)) {
```

Comment:
The asset error inconsistent size seems to be related to isSubtype method call here.